src/net.cpp

```diff
@@ -1575,6 +1575,19 @@ int CConnman::GetExtraBlockRelayCount() const
     return std::max(block_relay_peers - m_max_outbound_block_relay, 0);
 }
 
+std::vector<Network> CConnman::GetRelevantFixedSeedNetworks() const
+{
+    std::vector<Network> networks{};
+    for (int n = 0; n < NET_MAX; n++) {
+        enum Network net = (enum Network)n;
+        if (net == NET_UNROUTABLE || net == NET_INTERNAL) continue;
+        if (IsReachable(net) && addrman.size(net, std::nullopt) == 0) {
+            networks.push_back(net);
+        }
+    }
+    return networks;
+}
```

Comment:
This does not need to be a method of `CConnman` because it does not access any of its members (other than `addrman`). Would it be more appropriate to have it as `AddrMan` method? Then the public interface of `AddrMan` need not be changed by this PR.

Reply:
I'm not sure if this belongs into AddrMan because of the `IsReachable` - I think the information which networks are reachable belongs into `net`, and possibly at some later point into `CConnman` (I know `vfLimited` is currently a global, but this might change). I don't think it makes sense that `AddrMan` should know or deal with this information, so I think I prefer leaving this out of `AddrMan` and reducing the change of the public interface by combining the `size` functions as suggested by Amiti below.

Reply:
What about a standalone function?

Reply:
I'd be ok with that, but then we'd have to pass it addrman, which is a member of `CConnman`, so I'm not sure why that is better.